Channel.h

```diff
@@ -158,15 +159,21 @@ class Channel {
 
   void firstinit () {
     storage().clearData(address(),size());
-    List1Type cl = getList1();
-    cl.defaults();
+    List1Type cl1 = getList1();
+    List2Type cl2 = getList2();
+    cl1.defaults();
+    cl2.defaults();
   }
 
   List1Type getList1 () const {
     // we start always with list1
     return List1Type(address());
   }
 
+  List2Type getList2 () const {
+    return List2Type(address());
```

Comment:
List2 has same address as List1 ???
We should place List2 direct after List1 - so code should

`return List2Type(address()+List1::size());`

Method peerAddress() should also take care of the List2::size(). Add 

`offset += List1::size() + List2::size();`

Reply:
The repeater has no List1 and uses no peering, and there is no other device using List2.

I made the changes you suggested